bgpd/bgpd.c

```diff
@@ -3967,6 +3971,12 @@ static void peer_flag_modify_action(struct peer *peer, uint32_t flag)
 				peer_nsf_stop(peer);
 
 			UNSET_FLAG(peer->sflags, PEER_STATUS_PREFIX_OVERFLOW);
+
+			/* If we enter "no neighbor <neigh> shutdown",
+			 * reset received keepalives counter.
+			 */
+			peer->rtt_keepalive_rcv = 0;
```

Comment:
We need to reset  rtt_keepalive_rcv when peer goes down for any other reasons as well?

Reply:
Makes sense, I'll double-check.